### PR TITLE
[iOS] Tokenize Chip and Input

### PR DESCRIPTION
**Pull request by ayush547**
### Platforms Impacted
- [x] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes

Added tokens for Chip and Input
Enabled them in tester app
Fixed font token not applying in Input

### Verification

Chip normal -
image

Chip selected - 
image

Chip with close button - 
image

Input - 
image

Input Error state -
image

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/components/Input/src/Input.tsx [hunk 1]
```diff
@@ -22,16 +18,7 @@ import { useInput } from './useInput';
...
-  return (
-    (layer === 'focused' && state.focused && !state.text) ||
-    (layer === 'filled' && !state.focused && !!state.text) ||
-    (layer === 'typing' && state.focused && !!state.text) ||
-    userProps[layer] ||
-    (layer === 'hasIcon' && userProps.defaultIcon) ||
-    (layer === 'error' && !!userProps.error && !!state.text)
-  );
-};
+export const inputLookup = () => null;
```
Comment (Saadnajmi) [inline packages/components/Input/src/Input.tsx:21]: ?

File: packages/components/Input/src/Input.tsx [hunk 2]
```diff
@@ -48,60 +35,9 @@ export const Input = compose<InputType>({
...
-                </Slots.accessoryIconPressable>
-              )}
-            </Slots.input>
-          </IconWrapper>
-          {(assistiveText || error != undefined) && <Slots.assistiveText>{error ? error : assistiveText}</Slots.assistiveText>}
-        </Slots.root>
-      );
+  useRender: (_userProps: InputProps, _useSlots: UseSlots<InputType>) => {
+    return (_final: InputProps) => {
+      return null;
```
Comment (Saadnajmi) [inline packages/components/Input/src/Input.tsx:40]: ?

File: packages/components/Input/src/InputTokens.ios.ts [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { InputTokens } from './Input.types';
+
+export const defaultInputTokens: TokenSettings<InputTokens, Theme> = (t: Theme) =>
+  ({
+    iconSize: 24,
+    hasIcon: {
+      //spacingIconContent + iconSize
```
Comment (Saadnajmi) [inline packages/components/Input/src/InputTokens.ios.ts:11]: Remove commented out bit?

File: packages/components/Input/src/InputTokens.ios.ts [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      spacingAssistiveTextStart: globalTokens.size160 + 24,
+      spacingLabelStart: globalTokens.size160 + 24,
```
Comment (Saadnajmi) [inline packages/components/Input/src/InputTokens.ios.ts:13]: Should we move 24 to a constant?

File: packages/components/Input/src/Input.mobile.tsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
+/** @jsxRuntime classic */
```
Comment (rohanprasadofficial) [inline packages/components/Input/src/Input.mobile.tsx:1]: If Input is not planned to Desktop platform we can have only one file as Input.tsx and that should be fine.

File: packages/components/Input/src/Input.tsx (hunk 1 above)
Comment (ayush547) [reply packages/components/Input/src/Input.tsx:21]: Disabling for other platforms similar to Chip

File: packages/components/Input/src/Input.tsx (hunk 2 above)
Comment (ayush547) [reply packages/components/Input/src/Input.tsx:40]: Disabling for other platforms similar to Chip

File: packages/components/Input/src/InputTokens.ios.ts (hunk 3 above)
Comment (ayush547) [reply packages/components/Input/src/InputTokens.ios.ts:11]: Have added this to explain the value (In figma it is two separate values which translate to one single here)

File: packages/components/Input/src/InputTokens.ios.ts (hunk 4 above)
Comment (ayush547) [reply packages/components/Input/src/InputTokens.ios.ts:13]: All places that use icon size currently use hard coded values

Comment (ayush547) [reply packages/components/Input/src/InputTokens.ios.ts:13]: Should possibly be replaced everywhere though, not including it here

File: packages/components/Input/src/Input.mobile.tsx (hunk 5 above)
Comment (ayush547) [reply packages/components/Input/src/Input.mobile.tsx:1]: Chip was in a similar situation, it was recommended there to move it out to a mobile file, following same precedence here.

**Comment by rurikoaraki** [comment]
Side effect of this is that Input will render nothing for non-mobile platforms. Is that ok @saadnajmi?

**Comment by Saadnajmi** [comment]
> Side effect of this is that Input will render nothing for non-mobile platforms. Is that ok @Saadnajmi?

Yeah , I guess that's the standard we've kinda coalesced on

**Review by rurikoaraki — APPROVED** [review]

**Pull request merged**